MetasysServices/MetasysClient.cs

```diff
@@ -124,7 +125,8 @@ public int Timeout
                 if (Spaces != null) { Spaces.Version = version.Value; }
                 if (Streams != null) { Streams.Version = version.Value; }
                 if (Trends != null) { Trends.Version = version.Value; }
-                base.Version = value;
+                // override the base version as well
+                base.Version = version.Value;

```

Comment:
make this assignment consistent with all of the others which use `version.Value` on the RHS. Not a big deal but I noted that line 127 which I introduced in #147 (I think) just didn't follow the pattern.